src/Workspaces/Core/Portable/Workspace/Workspace_SourceGeneration.cs

```diff
@@ -56,72 +56,5 @@ await this.SetCurrentSolutionAsync(
             cancellationToken).ConfigureAwait(false);
 
         return;
-
-        static SourceGeneratorExecutionVersionMap GetUpdatedSourceGeneratorVersions(
```

Comment:
moved to helper used both here (when we hear about saves/builds), and by the compilation-state instance wqhen it is told that analyzer references changed.